examples/decoding/plot_haxby_decoder.py

```diff
@@ -0,0 +1,59 @@
+"""
+Decoding with decoder metaestimator: face vs house recognition
+==============================================================
+
+Here is a simple example of decoding with a the decoder metaestimator,
+reproducing the Haxby 2001 study on a face vs house discrimination task.
+"""
+
```

Comment:
You could add reference of Haxby 2001 here. It will be easy for users to follow.